CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.9.2 (UNRELEASED)
+--------------------------
+
+- Changes workflow scheduler to report scheduling errors to the user.
```

Comment:
What about expanding the news item a little bit::

- Changes workflow scheduler logging behaviour to also report the main reason behind scheduling errors to the users.